packages/core/package.json

```diff
@@ -6,6 +6,11 @@
   "main": "dist/index.js",
   "module": "dist/index.js",
   "types": "dist/index.d.ts",
+  "exports": {
```

Comment:
export breaks links between tests

`import { randomInt } from '@cognite/sdk-core/src/__tests__/testUtils';

but export fix the core error in powerops